zhmccli/_cmd_partition.py

```diff
@@ -916,6 +967,133 @@ def cmd_partition_console(cmd_ctx, cpc_name, partition_name, options):
             "{exc}: {msg}".format(exc=exc.__class__.__name__, msg=exc))
 
 
+def cmd_partition_dump(cmd_ctx, cpc_name, partition_name, **options):
+    # pylint: disable=missing-function-docstring
+
+    client = zhmcclient.Client(cmd_ctx.session)
+    partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
+
+    name_map = {
+        # These parameters are the same for CPCs with and without the storage
+        # management feature. Feature-specific parameters are added further
+        # down:
+        'configuration': 'dump-configuration-selector',
+        'parameters': 'dump-os-specific-parameters',
+        'lba': 'dump-record-lba',
+
+        # Options that are handled in this function:
+        'volume': None,
+        'timeout': None,
+    }
+    options = original_options(options)
+    properties = options_to_properties(options, name_map)
+    volume_opt = options['volume']  # It is required
+    timeout = options['timeout']  # It is optional but has a default
+
+    if storage_management_feature(partition):
+
```

Comment:
Is this blank line inteded?

Reply:
Yes, for better readability because the next line comment belongs to the code after it.